### ISSUE TITLE
aria-required-children should map to ACT rule ARIA required context role (ff89c9)

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3789]
In order to avoid duplicate issues, axe-core's aria-required-parent rule does not fail the following, and leaves the testing of this up to aria-required-children.

```html
<div role="list">
  <div aria-live="polite">
    <div role="listitem">item</div>
  </div>
</div>
```

To ensure correct mapping with ACT though, we need to add ARIA required context role (ff89c9) to the aria-required-children rule.

**Comment by padmavemulapati** [comment]
Validated with the 'axe-core' develop branch code base,
test snippet:
```
<div role="list">
  <div aria-live="polite">
    <div role="listitem">item</div>
  </div>
</div>
```

`aria-required-parent` is not failing for the test script, but failing for `aria-required-children` 



![Image](https://user-images.githubusercontent.com/47409579/203472160-7e553d9c-7c0b-45a6-9307-fe70aad2f34a.png)
